inst/stan/wwinference.stan

```diff
@@ -111,23 +113,26 @@ transformed data {
 
 // The parameters accepted by the model.
 parameters {
-  vector[n_weeks-1] w; // weekly random walk of state-level mean baseline R(t) (log scale)
+  vector[n_weeks-1] w; // weekly random walk of ref pop baseline R(t) (log scale)
   real<lower=0> eta_sd;
+  real m; // log scale scalar shift of reference population from central dynamic
+  real m_r; // shift of unobserved initial exponential growth rate of reference pop from central dynamic
+  real m_first_obs; // shift of unbserved initial infections per capita of reference pop from central dynamic
   real<lower=0, upper=1> autoreg_rt;// coefficient on AR process in R(t)
-  real log_r_mu_intercept; // state-level mean baseline reproduction number estimate (log) at t=0
+  real log_r_0_intercept; // reference subpop baseline reproduction number estimate (log) at t=0
   real<lower=0> sigma_rt; // magnitude of site level variation from state level
-  real<lower=0, upper=1> autoreg_rt_site;
+  real<lower=0, upper=1> autoreg_rt_subpop;
   real<lower=0, upper=1> autoreg_p_hosp;
-  matrix[n_subpops, n_weeks] error_site; // matrix of subpopulations
+  matrix[include_ww ? n_subpops-1 : 1, n_weeks] error_subpop;
```

Comment:
I think this has to be 1 because of the initialization (you can't initialize a [0,0] matrix).... it shouldn't get used.

Reply:
In that case use an array of vectors?

Reply:
Yeah I have never found a good solution to this other than setting it to not be 0 and not using it. It works ok for vectors but not for matrices because a [0,0] matrix in R gets passed to stan as a length 0 vector... recognize this as a hacky work around so happy to hear otherwise if you are able to get something to work for the initialization.

Reply:
pass `NULL`?

Reply:
I need to play around with options but I think stan still give you a warning if an initial value is missing even for a parameter specified as [0,0]